addons/mail/static/src/core/message_model.js

```diff
@@ -171,11 +174,12 @@ export class Message {
         return `${url("/web")}#model=${this.resModel}&id=${this.id}`;
     }
 
-    get isBodyEmpty() {
-        return (
-            !this.body ||
-            ["", "<p></p>", "<p><br></p>", "<p><br/></p>"].includes(this.body.replace(/\s/g, ""))
-        );
+    get editDate() {
+        return this.write_date !== this.date ? this.write_date : false;
```

Comment:
Shouldn't this compare to `create_date` rather than `date`?

All demo messages on general have edit date even though they were never edited, same for chatter messages. This might be related to this suggestion.

`date` is "unreliable" user input (especially when coming from email) whereas `create_date` and `write_date` are both technical (and thus more controlled).